specification/monitor/resource-manager/readme.md

```diff
@@ -31,17 +31,53 @@ title: MonitorClient
 ``` yaml
 description: Monitor Management Client
 openapi-type: arm
-tag: package-preview-2023-04
+tag: package-2023-04
 directive:
   - suppress: Example Validations
     reason: 'There are open issues (bugs) in the validator affecting some of the examples and since there is no way to selectively disable the validation for a particular example or paths, all of the example validation is being turned off.'
 ```
 
+
+### Tag: package-2023-04
+
+These settings apply only when `--tag=package-2023-04` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-04'
+input-file:
+  - Microsoft.Monitor/stable/2023-04-03/monitoringAccounts_API.json
+  - Microsoft.Monitor/stable/2023-04-03/operations_API.json
+  - Microsoft.Insights/stable/2022-10-01/autoscale_API.json
+  - Microsoft.Insights/stable/2015-04-01/operations_API.json
+  - Microsoft.Insights/stable/2016-03-01/alertRulesIncidents_API.json
+  - Microsoft.Insights/stable/2016-03-01/alertRules_API.json
+  - Microsoft.Insights/stable/2016-03-01/logProfiles_API.json
+  - Microsoft.Insights/preview/2021-05-01-preview/diagnosticsSettings_API.json
+  - Microsoft.Insights/preview/2021-05-01-preview/diagnosticsSettingsCategories_API.json
+  - Microsoft.Insights/stable/2023-01-01/actionGroups_API.json
+  - Microsoft.Insights/preview/2023-03-01-preview/tenantActionGroups_API.json
+  - Microsoft.Insights/stable/2015-04-01/activityLogs_API.json
+  - Microsoft.Insights/stable/2015-04-01/eventCategories_API.json
+  - Microsoft.Insights/stable/2015-04-01/tenantActivityLogs_API.json
+  - Microsoft.Insights/stable/2021-05-01/metricDefinitions_API.json
+  - Microsoft.Insights/stable/2021-05-01/metrics_API.json
+  - Microsoft.Insights/stable/2019-03-01/metricBaselines_API.json
+  - Microsoft.Insights/stable/2018-03-01/metricAlert_API.json
+  - Microsoft.Insights/preview/2022-08-01-preview/scheduledQueryRule_API.json
+  - Microsoft.Insights/preview/2017-12-01-preview/metricNamespaces_API.json
+  - Microsoft.Insights/preview/2018-11-27-preview/vmInsightsOnboarding_API.json
+  - Microsoft.Insights/preview/2021-07-01-preview/privateLinkScopes_API.json
+  - Microsoft.Insights/stable/2020-10-01/activityLogAlerts_API.json
+  - Microsoft.Insights/stable/2022-06-01/dataCollectionEndpoints_API.json
+  - Microsoft.Insights/stable/2022-06-01/dataCollectionRuleAssociations_API.json
+  - Microsoft.Insights/stable/2022-06-01/dataCollectionRules_API.json
+  - Microsoft.Monitor/preview/2021-06-03-preview/monitoringAccounts_API.json
+  - Microsoft.Monitor/preview/2021-06-03-preview/operations_API.json
```

Comment:
Duplicate with `stable/2023-04-03`, please double check.

Reply:
thanks Chenjie, removed this entry